src/libNode/Node.h

```diff
@@ -158,16 +158,16 @@ class Node : public Executable {
 
   // Fallback Consensus
   std::mutex m_mutexFallbackTimer;
-  uint32_t m_fallbackTimer;
+  uint32_t m_fallbackTimer{};
   bool m_fallbackTimerLaunched = false;
-  bool m_fallbackStarted;
+  bool m_fallbackStarted{};
```

Comment:
What does `{}` for `bool` default to?
And should the line immediately above this be `{false}`?
Curly brace-style initialization takes some getting used to.

Reply:
defaults to false